src/props.fonts.css

```
@@ -1,7 +1,22 @@
 :where(html) {
-  --font-sans: system-ui,-apple-system,Segoe UI,Roboto,Ubuntu,Cantarell,Noto Sans,sans-serif;
-  --font-serif: ui-serif,serif;
-  --font-mono: Dank Mono,Operator Mono,Inconsolata,Fira Mono,ui-monospace,SF Mono,Monaco,Droid Sans Mono,Source Code Pro,monospace;
+  --font-system-ui: system-ui, sans-serif;
+  --font-transitional: Charter, Bitstream Charter, Sitka Text, Cambria, serif;
```

```suggestion
  --font-transitional: ui-serif, Charter, Bitstream Charter, Sitka Text, Cambria, serif;
```
this will aid in less side effects in the update

---

I definitely understand not wanting side effects, but I feel that this change loses the benefit of this font stack over just using ui-serif. It's my understanding that transitional fonts are a specific type of serif font. Since ui-serif is first, it will likely always be chosen (where supported). From my perspective the other font families might as well not be there. Unless there's something I'm missing, I'm not a font expert 😅

---

just looked at the support for `ui-serif` and it's only Safari, so sounds like the only devices that will get it will be iPhones. it'll be the same as `ui-rounded` in the `rounded-sans` set.

---

Alright.  I suppose it's ok, since if you're using this font stack you don't care about the exact font.  it would bug me if this stack could choose a font that is not a transitional font (since not all serif fonts are transitional fonts) when a transitional font was available to the user

---

that seems fair,  but we're also doing this `--font-serif: var(--font-transitional);` so I guess we'd need a new set of values for font-serif that excludes the transitional serif fonts? that seems weird tho? looking at the set of fonts we have, that seems like the best one to drop it into yeah?

---

Could we do `--font-serif: ui-serif, serif;`? Since it'll be removed in the next major update I don't see the harm in the duplication

---

works for me!

---

Done!